symfony/templates/bundles/TwigBundle/Exception/error404.html.twig

```diff
@@ -0,0 +1,13 @@
+{% extends 'general/status.html.twig' %}
+{% block title %}
+    {{ status_code }} | {{ 'icapps.website.errors.404.title'|trans|raw }}
+{% endblock %}
+{% block message %}
```

Comment:
Exception pages hebben niets te maken met TwigBundle, right?

We steken templates onder /bundles om bundle related templates uit te breiden of overschrijven.

=> deze kunnen beter onder general, system of status